applications/StructuralMechanicsApplication/python_scripts/set_spherical_local_axes_process.py

```diff
@@ -0,0 +1,34 @@
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+import KratosMultiphysics.StructuralMechanicsApplication as SMA
+from KratosMultiphysics import Logger
+
+def Factory(settings, Model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
+    return SetSphericalLocalAxesProcess(Model, settings["Parameters"])
+
+class SetSphericalLocalAxesProcess(KM.Process):
+    def __init__(self, Model, settings):
+        KM.Process.__init__(self)
+        self.settings = settings;
+        self.model_part = Model[self.settings["model_part_name"].GetString()]
+
+    def ExecuteInitialize(self):
+        default_settings = KM.Parameters(
+            """{
+                "model_part_name"            : "set_model_part_name",
+                "spherical_reference_axis"   : [0.0,0.0,1.0],
+                "spherical_central_point"    : [0.0,0.0,0.0]
+            }""");
+
+        self.settings.ValidateAndAssignDefaults(default_settings)
+        KM.Process.__init__(self)
+        # Let's compute the local axes
+        Logger.PrintInfo("SetSphericalLocalAxesProcess:: ","Setting the oriented local axes...")
+        self.settings.RemoveValue("model_part_name")
+        SMA.SetSphericalLocalAxesProcess(self.model_part, self.settings).ExecuteInitialize()
```

Comment:
@AlejandroCornejo can you in this case do it like I did [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py#L37) i.e. return the Cpp process to make sure that all the fcts in the interface are called?

Reply:
I do not understand exactly why calling the `return`... in this case I am creating the cpp process and then calling directly the `initialize`. Do you mean that I should call the whole `Execute`?

Reply:
The idea is that you don't have the python process, but directly return the C++ version (check my link in detail)
Imagine that in the future you implement e.g. `ExecuteFinalize` in the C++ process. Then you would have to modify also the python process just for calling an additional method
If you directly return the C++ process it will be automatically called

Reply:
Then I'll have to implement only the `Factory`?

Reply:
yes exactly
sorry if I didn't make it clear